app/policies/thredded/post_policy.rb

```diff
@@ -26,7 +26,7 @@ def initialize(user, post)
     end
```

Comment:
Please replace the **`MessageboardPolicy`**`#post?` method with:

```ruby
    def post?
      @user.thredded_admin? ||
        (!@messageboard.locked? || moderate?) &&
        @user.thredded_can_write_messageboards.include?(@messageboard)
    end
```

Reply:
This is done, you commented on the wrong line.